docs/api/profiles.md

```diff
@@ -0,0 +1,86 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile, but a Profile is always created
+together with a User.
```

Comment:
I'd separate the profile creation from the user, i.e. require a separate api call. So a profile is not created together with a user.